build

```diff
@@ -126,9 +116,9 @@ cp_dyn_libs() {
         | sort -u)
 }
 
-## make_zip turns .tar.gz into a .zip with a slightly different name.
+## Re-pack the relx assembled .tar.gz to EMQ X's package naming scheme
 ## It assumes the .tar.gz has been built -- relies on Makefile dependency
-make_zip() {
+make_tgz() {
```

Comment:
If we are now building `tar.gz` files directly, is this function still needed?  Maybe it's sole purpose now is to copy the dynamic libraries into the tarball?

Reply:
yes, we still need to keep this re-pack.